parastell/source_mesh.py

```diff
@@ -95,7 +95,7 @@ def __init__(
 
         self.scale = m2cm
 
-        for name in kwargs.keys() & ('scale'):
+        for name in kwargs.keys() & ['scale']:
```

Comment:
Sometimes single strings are treated as lists... not sure if that's the problem, but if it is, I don't think this will fix it.

One fix is to add a dangling comma after the string, inside the list/tuple

Reply:
I can confirm that changing the tuple to a list fixes the issue.

Reply:
Adding a trailing comma also works though.

Reply:
I think I like the tuple better because of the way it communicates immutability.

Reply:
Fair enough, I've reverted to tuples but used a trailing comma for `SourceMesh`.